oscd.go

```diff
@@ -27,23 +29,44 @@ func main() {
 	}
 
 	targetDir := os.Args[1]
+
+	// get abs path if given relative
+	if !filepath.IsAbs(targetDir) {
+		td, err := filepath.Abs(targetDir)
+		if err != nil {
+			panic(err)
```

Comment:
panic() is like manually throwing an exception. I don't like using it unless I intend to recover (or i need defers to be executed)